infra/build/functions/deploy.sh

```diff
@@ -94,19 +102,30 @@ deploy_scheduler $BASE_IMAGE_SCHEDULER_JOB \
 				 "$BASE_IMAGE SCHEDULE" \
 				  $BASE_IMAGE_JOB_TOPIC \
 				  "$BASE_IMAGE_MESSAGE" \
-				  $PROJECT_ID
+				  $BASE_PROJECT_ID
+
+deploy_scheduler $COVERAGE_BUILD_SCHEDULER_JOB \
+				 "$COVERAGE_BUILD_SCHEDULE" \
+				 $COVERAGE_BUILD_JOB_TOPIC \
+				 "$COVERAGE_BUILD_MESSAGE" \
+				 $PROJECT_ID
 
 deploy_cloud_function sync \
-					  project_sync \
+					  sync \
```

Comment:
shouldn't this be `project_sync` as you renamed `sync.py` -> `project_sync.py`?

Reply:
This is just the display name that will be shown on the cloud platform, I'll chang it if required